packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -598,46 +600,90 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
               onMouseMove={this._onRectMouseMove}
               onMouseOut={this._onRectMouseOut}
               onMouseOver={this._onRectMouseMove}
+              {...(this.props.optimizeLargeData && {
+                'data-is-focusable': true,
+                role: 'img',
+                'aria-label': `${points[index].legend}, series ${index + 1} of ${points.length} with ${
+                  points[index].data.length
+                } data points.`,
+              })}
             />
           )}
         </React.Fragment>,
       );
     });
 
     const circleRadius = pointOptions && pointOptions.r ? Number(pointOptions.r) : 8;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    this._stackedData.forEach((singleStackedData: Array<any>, index: number) => {
-      if (points.length === index) {
-        return;
-      }
-      graph.push(
-        <g key={`${index}-dots-${this._uniqueIdForGraph}`} d={area(singleStackedData)!} clipPath="url(#clip)">
-          {singleStackedData.map((singlePoint: IDPointType, pointIndex: number) => {
+    if (!this.props.optimizeLargeData) {
+      // Render all data points for small dataset
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      this._stackedData.forEach((singleStackedData: Array<any>, index: number) => {
+        if (points.length === index) {
+          return;
+        }
+        graph.push(
+          <g key={`${index}-dots-${this._uniqueIdForGraph}`} d={area(singleStackedData)!} clipPath="url(#clip)">
+            {singleStackedData.map((singlePoint: IDPointType, pointIndex: number) => {
+              const circleId = `${this._circleId}_${index * this._stackedData[0].length + pointIndex}`;
+              const xDataPoint = singlePoint.xVal instanceof Date ? singlePoint.xVal.getTime() : singlePoint.xVal;
+              lineColor = points[index]!.color!;
+              return (
+                <circle
+                  key={circleId}
+                  id={circleId}
+                  data-is-focusable={true}
+                  cx={xScale(singlePoint.xVal)}
+                  cy={yScale(singlePoint.values[1])}
+                  stroke={lineColor}
+                  strokeWidth={3}
+                  fill={this._updateCircleFillColor(xDataPoint, lineColor, circleId)}
+                  onMouseOut={this._onRectMouseOut}
+                  onMouseOver={this._onRectMouseMove}
+                  onClick={this._onDataPointClick.bind(this, points[index]!.data[pointIndex].onDataPointClick!)}
+                  onFocus={() => this._handleFocus(index, pointIndex, circleId)}
```

Comment:
I am seeing 1 issue.
Click on a datapoint. Then hover on previous points. The hover card continues to render for the datapoint that was clicked earlier.